Kwartracker/Scenes/Home/HomeView.swift

```diff
@@ -28,6 +28,7 @@ struct HomeView: View {
 
         return SideBarStack(sidebarWidth: sideBarWidth, showSidebar: $showSidebar) {
             SideMenuView(selectedItem: $selectedMenu, showSidebar: $showSidebar)
+                .environmentObject(store)
```

Comment:
I think no need to add this because the [parent](https://github.com/jaimejazarenoiii/kwartracker-ios/blob/feature/side-menu-fetch/Kwartracker/App%20Main/KwartrackerApp.swift#L24) already has it, I am not sure but please check 😅